subscription-service/src/main/resources/application.properties

```diff
@@ -30,7 +30,12 @@ application.authentication.enabled = false
 # application.graylog.host = localhost
 # application.graylog.port = 12201
 # application.graylog.source = stellio-int
-
 # Pagination config for query resources endpoints
-application.pagination.limit-default = 30
-application.pagination.limit-max = 100
+application.pagination.limit-default=30
+application.pagination.limit-max=100
+#configure ressource for authentification
+spring.security.oauth2.client.registration.keycloak.authorization-grant-type=client_credentials
+spring.security.oauth2.client.registration.keycloak.client-id=senseen-gael
+spring.security.oauth2.client.registration.keycloak.client-secret=221d117c-b7fd-4b8d-8064-f9c51605c81c
```

Comment:
this is very bad! (btw, I removed all your sessions and changed your secret...)

Reply:
Can I create a KC account ?

Reply:
the problem is that you commited a secret on a public repository. you can still use the same client, just don't publish this info in a public place.

Reply:
I understood but how can I put the secret on the application ?

Reply:
Secrets are never stored in the source code. They are specific to each deployment and injected at runtime.

Reply:
so I can take off this line `spring.security.oauth2.client.registration.keycloak.client-secret` ?

Reply:
leave the property (that also makes a sort of referential of supported properties) but give it a default dumb value (same for client-id btw)

Reply:
Idem for token-uri ? It change to each deployment ?